src/main/webapp/WEB-INF/tags/instructor/courseStudentDetailsEdit/studentInformationTable.tag

```diff
@@ -0,0 +1,66 @@
+<%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<%@ attribute name="studentInfoTable" type="teammates.ui.template.StudentInfoTable" required="true" %>
+<%@ attribute name="newEmail" required="true" %>
+<%@ tag import="teammates.common.util.Const" %>
+<div class="panel panel-primary">
+    <div class="panel-body fill-plain">
+        <form action="<%=Const.ActionURIs.INSTRUCTOR_COURSE_STUDENT_DETAILS_EDIT_SAVE%>" method="post" class="form form-horizontal">
+            <input type="hidden" name="<%=Const.ParamsNames.COURSE_ID%>" value="${studentInfoTable.course}">
+            <div class="form-group">
+                <label class="col-sm-1 control-label">Student Name:</label>
+                <div class="col-sm-11">
+                    <input class="form-control" name="<%=Const.ParamsNames.STUDENT_NAME%>" 
+                           id="<%=Const.ParamsNames.STUDENT_NAME%>"
+                           value="${studentInfoTable.name}">
+                </div>
+            </div>
+            <c:if test="${studentInfoTable.hasSection}">
+                <div class="form-group">
+                    <label class="col-sm-1 control-label">Section Name:</label>
+                    <div class="col-sm-11">
+                        <input class="form-control" name="<%=Const.ParamsNames.SECTION_NAME%>" 
+                               id="<%=Const.ParamsNames.SECTION_NAME%>"
+                               value="${studentInfoTable.section}">
+                    </div>
+                </div>
+            </c:if>
+            <div class="form-group">
+                <label class="col-sm-1 control-label">Team Name:</label>
+                <div class="col-sm-11">
+                    <input class="form-control" name="<%=Const.ParamsNames.TEAM_NAME%>" 
+                           id="<%=Const.ParamsNames.TEAM_NAME%>"
+                           value="${studentInfoTable.team}">
+                </div>
+            </div>
+            <div class="form-group">
+                <label class="col-sm-1 control-label">E-mail Address:
+                    <input type="hidden" name="<%=Const.ParamsNames.STUDENT_EMAIL%>" 
+                           id="<%=Const.ParamsNames.STUDENT_EMAIL%>"
+                           value="${studentInfoTable.email}">
+                </label>
+                <div class="col-sm-11">
+                    <input class="form-control" name="<%=Const.ParamsNames.NEW_STUDENT_EMAIL%>" 
+                           id="<%=Const.ParamsNames.NEW_STUDENT_EMAIL%>"
+                           value="${newEmail}">
+                </div>
+            </div>
+            <div class="form-group">
+                <label class="col-sm-1 control-label">Comments:</label>
+                <div class="col-sm-11">
+                    <textarea class="form-control" rows="6" name="<%=Const.ParamsNames.COMMENTS%>" 
+                              id="<%=Const.ParamsNames.COMMENTS%>">${studentInfoTable.comments}</textarea>
```

Comment:
I recall there is a reason for no whitespace in between the `textarea` opening and closing tags. If that's the case can you add comment for it?